source/reference/operator/aggregation/filter.txt

```diff
@@ -41,10 +41,10 @@ Definition
 
       * - ``as``
 
-        - The variable name for the element in the ``input`` array. The
-          ``as`` expression accesses each element in the ``input``
-          array by this :doc:`variable
-          </reference/aggregation-variables>`.
+        - Optional. The variable name for the element in the ``input`` array. The
+          ``cond`` expression accesses each element in the ``input`` array
+          by this :doc:`variable </reference/aggregation-variables>`.
+          If not specified, ``as`` defaults to ``$$this``.
```

Comment:
Not sure, but should it be "this"? since in the cond expression, we'll access the variable as $$this, so the value of "as" would be "this".